src/testrel.jl

```diff
@@ -184,6 +184,10 @@ function Step(;
     )
 end
 
+ReadQuery(s::String; kwargs...) = Step(; query=s, readonly=true, kwargs...)
```

Comment:
These are quite simple, but I think a comment to explain their purpose would help anyone learning to use test_rel. Something like:
```suggestion
# Helpers for commonly used Steps
ReadQuery(s::String; kwargs...) = Step(; query=s, readonly=true, kwargs...)
```